data/events/scripts/player.lua

```diff
@@ -2,6 +2,7 @@ function Player:onBrowseField(position)
 	if hasEventCallback(EVENT_CALLBACK_ONBROWSEFIELD) then
 		return EventCallback(EVENT_CALLBACK_ONBROWSEFIELD, self, position)
 	end
+
```

Comment:
I also dont like those new lines.

Reply:
Yes, is stranger visually. Mainly with just an 'if'.
The intention of adding a space is so that when contributing, and when there is a change, the code looks like this [instead](https://github.com/otland/forgottenserver/blob/f5ebb54e0af792d29332001c06f28456cafb6bf3/data/movements/scripts/swimming.lua#L24). And not [that](https://github.com/otland/forgottenserver/blob/f5ebb54e0af792d29332001c06f28456cafb6bf3/data/movements/scripts/swimming.lua#L12). Visually it's better for me, but it depends on you.

Following the [guide](https://github.com/otland/forgottenserver/wiki/TFS-Coding-Style-Guide#return-values), you are correct.

Reply:
I don't see anything related to Lua in that guide

Reply:
Not really, just c++

Reply:
here you have a real guide on how to format the style in lua
http://lua-users.org/wiki/LuaStyleGuide

I know you prefer one thing than the other, but if we are to the case everyone wants different things and we must follow rules, because if you like to add extra spaces in places where it is intended not to encode anything, then in the future someone else I could say: well I like to add 3 extra lines and it is what I prefer and who against his word?

It is not a complaint, I love this scripting language very much and I would like it to keep its original style, if you use a small script like this:
```lua
local count = 0
debug.sethook(function (...)
	count = count +1
	print(count)
end, "l")
```
You will notice that these extra lines are counted, so it seems that they consume at least one tick when reading the code blocks, Okay, I accept it, 1 extra line does not seem to be something very serious :D

I hope someone else can share their opinions, you have done a great job in formatting thousands of lines of code ;)

Reply:
Yes, I understand.
I am waiting for the decision. Do I maintain or withdraw these changes? So, I have no problem with that.

- Should I keep no spaces [here (fn return)](https://github.com/otland/forgottenserver/pull/3414/files#diff-2d4610e4f2008206f76364306f5d6d1d9720def117d123997fb1ecd5ed34aef2R23) or [here (fn end)](https://github.com/otland/forgottenserver/pull/3414/files#diff-45e836f9c478222a9442b880a9ae8b2eed04317c5544e8416c14ee8161575768R7)?
- Should put spaces [here](https://github.com/otland/forgottenserver/pull/3414/files#diff-b579492fb6f0f01ecd7af200684d0592b704a96a5296cb491da85d38c928836cR63)?


With these two questions, I must finish lua code.

I liked the link.

Reply:
Hopefully someone else comments something about it, I think that all the files have not yet been completely reviewed

As for me, I would like these extra lines not to be added in the files in the folder: `data/events/scripts/`
I don't really care about the other files if they change their style, I'll do an extra check on all the lua files that you modifed and I'll do one more check

Reply:
I will maintain the standard of [LuaStyleGuide](http://lua-users.org/wiki/LuaStyleGuide).